sentry/src/main/java/io/sentry/SentryEnvelopeItem.java

```diff
@@ -151,6 +153,50 @@ public static SentryEnvelopeItem fromUserFeedback(
     return new SentryEnvelopeItem(itemHeader, () -> cachedItem.getBytes());
   }
 
+  public static SentryEnvelopeItem fromAttachment(
+      @NotNull ILogger logger, final @NotNull Attachment attachment) {
+
+    final CachedItem cachedItem =
+        new CachedItem(
+            () -> {
+              if (attachment.getBytes() != null) {
+                return attachment.getBytes();
+              } else if (attachment.getPathname() != null) {
+                try (FileInputStream fileInputStream =
```

Comment:
M: should we do a safe check for `isFile()` and `canRead()`? it's probably cheaper than `SecurityException`